assets/constants.js

```diff
@@ -443,5 +443,7 @@ var constants = {
 
   "FEEDBACK_SUBJECT_MAX_CHAR_LIMIT": 50,
 
-  "USE_NEW_SUGGESTION_FRAMEWORK": false
+  "USE_NEW_SUGGESTION_FRAMEWORK": false,
+
+  "ENABLE_GCS_STORAGE_FOR_IMAGES": false
```

Comment:
This is not used anywhere.

Reply:
Yes it is not right now, it will be used in the fetching and uploading of images from GCS PR.

Reply:
Okay, will add that in the next PR